components/operator.py

```diff
@@ -17,91 +15,105 @@ class OperatorWidget:
     ]  # Made up for testing purposes
 
     @staticmethod
-    def operator_widget():
+    def get_component():
         widget = dbc.FormGroup(
             children=[
                 dbc.Label("Operatör"),
-                OperatorWidget._operator_radiobuttons(),
-                OperatorWidget._operator_dropdown(),
+                Operator._operator_radioitems(),
+                Operator._operator_collapse(),
             ],
         )
         return widget
 
     @staticmethod
-    def _operator_radiobuttons():
+    def _operator_radioitems():
```

Comment:
Either change this back to radiobuttons or change line 16 in care_type.py and line 19 in municipalities.py to radioitems (and any other places where it's wrong)

Reply:
Fixed in 4c8a45ad45748c7f274e83dbcd0c06abcf030bef